EDU-306: How to integrate SAML with Temporal Cloud (#1966)
temporalio/documentation: docs-src/cloud-context/saml-finish.md

---
id: saml-finish
title: How to finish your SAML configuration
sidebar_label: Finish SAML configuration

> description: To finish your SAML configuration, send the sign-in URL and certificate to us and then test your connection.

Reviewer: They need to include the email domain they want to map to their saml connection. i.e. for us it is "temporal.io".

They have the option to provide us multiple domains.

Reviewer: nit: might want to name certificate the "X509 Signing Certificate"

tags:
  - guide-context
---

After you configure SAML with your IdP, we can finish the configuration on our side.
To do that, send the following information to us:

- The sign-in URL from your application

> - The SAML sign-in certificate

Reviewer: Please include the "IdP Domain" as a requirement--and note they can provide multiple

When you receive confirmation from us that we have finished configuration, log in to Temporal Cloud.
This time, though, enter your email address in **Enterprise identity** and select **Continue**.
Do not select **Continue with Google** or **Continue with Microsoft**.

> You will be redirected to the authentication page of your IdP.

Reviewer: So--since the doc is more long-lived idk if we should add this but, the customer needs to let us know if they want to convert their account to be SAML, which means
1. their user invites will be SAML only
2. they can only login via SAML (google sso and microsoft login will be rejected)

However I can see this functionality changing in the future so maybe it's okay to leave it out of the docs

Author: Let's stet for now. This would be good to add later if/when needed.

---
id: saml-finish
title: How to finish your SAML configuration
sidebar_label: Finish SAML configuration
description: To finish your SAML configuration, send us the sign-in URL, X.509 certificate, and IdP domains and then test your connection.
tags:
  - guide-context
---

After you configure SAML with your IdP, we can finish the configuration on our side.

> To do that, [create a support ticket](/cloud/how-to-create-a-ticket-for-temporal-support) and send the following information to us:

Reviewer 2: ```suggestion
[Create a support ticket](/cloud/how-to-create-a-ticket-for-temporal-support) that includes the following information:
```

- The sign-in URL from your application
- The X.509 SAML sign-in certificate

> - At least one IdP domain to map to the SAML connection

Reviewer 2: ```suggestion
- One or more IdP domains to map to the SAML connection
```

> Generally, your IdP domain is the same as the domain for your email address.

Reviewer 2: ```suggestion
Generally, the provided IdP domain is the same as the domain for your email address.
```

> You can provide multiple IdP domains.

Reviewer 2: May not need this if the IdP bullet point is reworded

Author: I'm inclined to keep it to confirm to the reader that, yes, they can indeed provide more than one.
